src/App.test.tsx

```diff
@@ -1,9 +1,50 @@
 import React from "react";
 import { render, screen } from "@testing-library/react";
 import App from "./App";
+import { QuestForest } from "./quest-tree";
+import { MissionName } from "./types";
 
 test("renders learn react link", () => {
```

Comment:
We can remove this test, it's boilerplate from create-react-app that has no use at this point.